cves/CVE-2007-0404.yml

```diff
@@ -49,27 +49,39 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: A script for compliling messages was made vulnerable because the inputs
+  submitted were not being checked to make sure that they were not malicious in nature.
+  The script job was to make OS level commands and if the inputs were not checked correctly
```

Comment:
Did you mean "The script's job"?

Reply:
OS-level commands

Reply:
This sentence is trying to say a bit too much and becomes hard to read after the second or third clause.  Try breaking it up into 2-3 sentences.